src/quo/components/community/style.cljs

```diff
@@ -133,18 +133,24 @@
                       colors/neutral-80
                       theme)})
 
-(defn channel-action-touch
-  [big?]
-  {:flex      1
-   :max-width (if big? 216 104)})
+(defn channel-action-container
+  [{:keys [big? disabled?]}]
+  (cond-> {:height 102}
+    disabled?  (assoc :opacity 0.3)
+    ;; NOTE: The big action has to fill the available space when there's a small
+    ;; action next to it. If alone, take an approx. proportion of space ~2/3.
+    ;; Using fixed sizes according to the designs will make it look wrong on
+    ;; all other devices.
```

Comment:
Hi, Thank you for fixing the issue

> If alone, take an approx. proportion of space ~2/3.

Please can you provide reference for this. Means a use case where we have it alone?

Reply:
Also, can something like this work?
```
(cond-> {:height 102 :flex 1}
    disabled?  (assoc :opacity 0.3)
    (not big?) (assoc :max-width 104)
```

Reply:
no specific use-cases, but up until a while back, there was only one big action for the pinned messages and no mute action in the chat, but it had a fixed size back then. 

Your example will definitely work, except that the big one alone will always take up the entire space, which might look weird when testing it in the preview :man_shrugging:

Reply:
maybe I can add a prop to `channel-action` for fixed/dynamic size and `channel-actions` will pass it as fixed if it's only one

Reply:
Ok, in that case, can we use something like

```
(cond-> {:height 102 :flex 1:max-width "Add fixed width here"}
    disabled?  (assoc :opacity 0.3)
    (not big?) (assoc :max-width 104)
```

Reply:
The thing is, I don't know what `:max-width` should be. If we use the one from the designs, it will be smaller on some devices e.g. on the pixel devices we use the big button doesn't take up the entire space when used with 2 actions

<img src="https://github.com/status-im/status-mobile/assets/21865759/05a5b664-a17d-4f16-b545-db686b746b16"  width=200/>

Reply:
simplified it and will see how the design review goes :smiling_face_with_tear: